.github/workflows/buildimages-update.yml

```diff
@@ -100,3 +104,43 @@ jobs:
           # the action fetches all branches and tags, in our case the branches we care about are already fetched
           # if they exist, so we can skip the fetch
           skip_fetch: true
+
+      - name: Check if PR exists
+        id: check_pr
+        env:
+          GH_TOKEN: ${{ github.token }}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Does this exist ? In the CI we're usually using `GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}`. I hope that this won't print the token as well

Reply:
[Nevermind !](https://stackoverflow.com/a/66903734)